### DISCUSSION TITLE
"ReferenceError: cordova is not defined" in Vue.js app

### DISCUSSION THREAD

**Discussion by tesicg** [quasarframework/quasar#10760] [General - Components / Directives / etc] [answered]
We have mobile Vue.js app that uses Quasar framework and Cordova. I'm trying to use Cordova object in Vue component and the code looks as following:

    <script>
    let permissions = cordova.plugins.permissions

    export default {
      ...
    }

In this case I get the following error:

    ReferenceError: cordova is not defined

If I write:

    let permissions = window.cordova.plugins.permissions

There's another error:

    TypeError: Cannot read properties of undefined (reading 'plugins')

My question is how to use Cordova in Vue component?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@tesicg 

Please post the `console.log(window.cordova)` output.
